src/components/OperationPanes/SetTrustLineFlags.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import OptionsTablePair from '../OptionsTable/Pair';
+import AssetPicker from '../FormComponents/AssetPicker.js';
+import PubKeyPicker from '../FormComponents/PubKeyPicker';
+import FlagfieldPicker from '../FormComponents/FlagfieldPicker';
+import CheckboxButtonPicker from "../FormComponents/CheckboxButtonPicker";
+import HelpMark from '../HelpMark';
+
+const flagsSet = {
+  'authorized': 'Authorized',
+  'authorizedToMaintainLiabilities': 'Authorized to maintain liabilites',
+};
+
+const flagsClear = {
+  'authorized': 'Authorized',
+  'authorizedToMaintainLiabilities': 'Authorized to maintain liabilites',
+  'clawbackEnabled': 'Clawback enabled',
+};
```

Comment:
Does it make sense to combine reused flags?

```
const DEFAULT_FLAGS = {
'authorized': 'Authorized',
  'authorizedToMaintainLiabilities': 'Authorized to maintain liabilites',
}

const flagsSet = {
...DEFAULT_FLAGS
}
etc.
```

Reply:
Keeping this for now for consistency. https://github.com/stellar/laboratory/issues/523